src/SFA.DAS.Assessor.Functions.Infrastructure/Configuration/AzureTableStorageConfigurationExtensions.cs

```diff
@@ -1,12 +1,12 @@
 using Microsoft.Extensions.Configuration;
 
-namespace SFA.DAS.Assessor.Functions.Infrastructure
+namespace SFA.DAS.Assessor.Functions.Infrastructure.Configuration
 {
     public static class AzureTableStorageConfigurationExtensions
     {
-        public static IConfigurationBuilder AddAzureTableStorageConfiguration(this IConfigurationBuilder builder, string connection, string appName, string environment, string version, string appStorageName)
+        public static IConfigurationBuilder AddAzureTableStorageConfiguration(this IConfigurationBuilder builder, string connection, string appName, string environment, string version)
```

Comment:
There is a DAS shared package for doing reading the azure table storage:

`sfa.das.configuration.azuretablestorage`

Example usage can be found in `das-reservations-jobs`

Reply:
Unfortunately I copied this from the 1094 branch - if I had known before I would have used it but I'd rather keep it the same as that branch for the merging